sesame/src/main/scala/org/w3/banana/sesame/extensions.scala

```diff
@@ -0,0 +1,42 @@
+package org.w3.banana
+package sesame
+
+import org.openrdf.model.Statement
+import org.openrdf.query.{GraphQueryResult, BindingSet, TupleQueryResult}
+import org.openrdf.repository.RepositoryResult
+
+/** small things to make life with Sesame easier */
+object extensions extends SesameResultsConversions
+
+
+trait SesameResultsConversions {
+
+  /**
+   * Implicit class that turns query result into iterator (so methods toList, map and so on can be applied to it)
+   *
+   * @param results results in sesame format
+   */
+  implicit class TupleResult(results: TupleQueryResult) extends Iterator[BindingSet]
```

Comment:
This really does not belong in this PR, and besides there are no tests. Besides, we already have https://github.com/w3c/banana-rdf/blob/5f88d33fbc890063ef1d507e1bcf7ca4d9c7efa1/rdf/common/src/main/scala/org/w3/banana/syntax/SolutionsSyntax.scala